mis_builder/readme/newsfragments/220.bugfix

```diff
@@ -0,0 +1 @@
+Mis builder export in XLSX should keep the percentage point suffix (pp) instead of replacing it with %
```

Comment:
"Keep the percentage point suffix (pp) instead of replacing it with %."

This will read better in the changelog.